hibike/devices/YogiBear/motor.h

```diff
@@ -0,0 +1,23 @@
+#ifndef MOTOR_H
+#define MOTOR_H
+
+//function prototypes
+void motorSetup();
+
+int readEnable();
+
+void enable();
```

Comment:
Changed and now the rest of the functions have names that reflect their purpose and imply which file it originates from. I have not, however, made their function names explicitly state the file it originates from